Integrations/IntezerV2/IntezerV2.py

```diff
@@ -0,0 +1,252 @@
+    handle_analyze_response(response)
+
+
+def make_analyze_by_file_request(file_id):
+    session = get_session()
+    file_data = demisto.getFilePath(file_id)
+    with open(file_data['path'], 'rb') as file_to_upload:
+        files = {'file': (file_data['name'], file_to_upload)}
+        return session.post(SERVER_URL + '/v2-0/analyze', files=files)
+
+
+def handle_analyze_response(response):
+    response = handle_response(response, ACCEPTABLE_HTTP_CODES)
+
+    result_url = response['result_url']
+    analysis_id = result_url.rsplit('/', 1)[-1]
+
+    context_json = {'Intezer.Analysis': {'ID': analysis_id, 'Status': 'Created', 'type':'File'}}
+
+    return_outputs('Analysis created successfully', context_json, response)
+
+
+def check_analysis_status_and_get_results():
+    analysis_type = demisto.args().get('analysis_type', 'File')
+    analysis_ids = argToList(demisto.getArg('analysis_id'))
+
+    for analysis_id in analysis_ids:
+        response = make_analysis_status_request(analysis_id, analysis_type)
+        analysis_result = handle_analysis_result(response)
+
+        if analysis_result and analysis_type == 'Endpoint':
+            enrich_dbot_and_display_endpoint_analysis_results(analysis_result, demisto.getArg('indicator_name'))
+        elif analysis_result and analysis_type == 'File':
+            enrich_dbot_and_display_file_analysis_results(analysis_result)
+
+
+def make_analysis_status_request(analysis_id, analysis_type):
+    result_url = get_result_url(analysis_type, analysis_id)
+    session = get_session()
+    return session.get(result_url)
+
+
+def get_result_url(analysis_type, analysis_id):
+    result_url = '{}/{}/{}{}'.format(SERVER_URL, API_VERSION,
+                                         'endpoint-analyses/' if analysis_type == 'Endpoint' else 'analyses/',
+                                         analysis_id)
+
+    return result_url
+
+
+def handle_analysis_result(response):
+    json_response = handle_response(response, ACCEPTABLE_HTTP_CODES)
+
+    if response.status_code != 200:
+        result_url = json_response['result_url']
+        analysis_id = result_url.rsplit('/', 1)[-1]
+
+        context_json = {'Intezer.Analysis(val.ID === obj.ID)': {'ID': analysis_id,
+                                                                'Status': 'InProgress'}}
+
+        return_outputs('Analysis is still in progress', context_json)
+        return
+
+    return json_response['result']
+
+
+def enrich_dbot_and_display_file_analysis_results(result):
+    verdict = result['verdict']
+    sha256 = result['sha256']
+    analysis_id = result['analysis_id']
+
+    dbot = {
+        'Vendor': 'Intezer',
+        'Type': 'hash',
+        'Indicator': sha256,
+        'Score': dbot_score_by_verdict.get(verdict, 0)
+    }
+
+    file = {'SHA256': sha256, 'Metadata': result, 'ExistsInIntezer': True}
+
+    if verdict == 'malicious':
+        file['Malicious'] = {'Vendor': 'Intezer'}
```

Comment:
`Malicious` should also contain `Description` which explains why is the IOC malicious.